server/src/main/java/org/apache/iotdb/db/engine/merge/task/MergeMultiChunkTask.java

```diff
@@ -401,9 +401,9 @@ private int writeRemainingUnseq(IChunkWriter chunkWriter,
   private int writeChunkWithUnseq(Chunk chunk, IChunkWriter chunkWriter, IPointReader unseqReader,
       long chunkLimitTime, int pathIdx) throws IOException {
     int cnt = 0;
-    ChunkReader chunkReader = new ChunkReaderWithoutFilter(chunk);
-    while (chunkReader.hasNextBatch()) {
-      BatchData batchData = chunkReader.nextBatch();
+    AbstractChunkReader AbstractChunkReader = new ChunkReader(chunk, null);
+    while (AbstractChunkReader.hasNextBatch()) {
+      BatchData batchData = AbstractChunkReader.nextBatch();
```

Comment:
The first letter o variable should be lowercase

Reply:
fixed